doozer/doozerlib/cli/images_streams.py

```diff
@@ -1174,6 +1201,12 @@ def check_if_upstream_image_exists(upstream_image):
 
                 exectools.cmd_assert(f'git add -f {str(ci_operator_config_path)}')
 
+                if desired_ci_build_root_image and 'golang' in desired_ci_build_root_image:
+                    try:
```

Comment:
Code looks good :+1:  Can we add a comment explaining why for this odd looking block, help future us and code readers